tests/test_basic.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import time
+import logging
+
+# logger
+logger = logging.getLogger(__name__)
+
+
+def test_discover_plugin():
```

Comment:
this is not really common dispatcher

Reply:
this is for the specific plugin

Reply:
exactly! all the others are common dispatcher, and exist in several places now.